### DISCUSSION TITLE
✓ Preparing to download box ✖ Downloading Unbox failed! Error: Truffle Box at URL https://github.com:truffle-box/pet-box doesn't exist. If you believe this is an error, please contact Truffle support.

### DISCUSSION THREAD

**Discussion by karandhumal2023** [ConsenSys-archive/truffle#5851] [Truffle Boxes]
✓ Preparing to download box
✖ Downloading
Unbox failed!
Error: Truffle Box at URL https://github.com:truffle-box/pet-box doesn't exist. If you believe this is an error, please contact Truffle support.

**Comment by FlaviuWeb3** [comment]
Hi @kd8805 please note that you have to use the following command: truffle unbox pet-shop

**Comment by karandhumal2023** [comment]
thanks bro finally its works
